integration-tests/Makefile

```diff
@@ -23,7 +23,8 @@ $(DEPLOYMENT_SRC_DIR).built/%:
 
 
 CONTAINER_FLAGS := --env-file  $(DEPLOYMENT_SRC_DIR)deployment.cfg --network veraison-net \
-		   -v $(THIS_DIR):/integration-tests
+		   -v $(THIS_DIR):/integration-tests \
+		   -v $(STORES_VOLUME):/opt/veraison/stores
```

Comment:
Question on line 29:

a) Where is `DEPLOYMENT_DEPS` used ?

b) If used, should we not append the management-container

Reply:
They're not, good catch. I think the intiention was for them to be used as the dependencies for the `.built/image` rule, but uses `$(DEPLOYMENT_SRC_DIR).built/deploy` instead, which encapsulates those (and would cover the new binary as well).

I'll remove.